packages/components/src/TokenName/TokenName.tsx

```diff
@@ -7,28 +7,51 @@ import { SvgIconProps } from '@material-ui/core/SvgIcon';
 
 import { TokenIcon } from '../TokenIcon/TokenIcon';
 
+type IconSizes = 'small' | 'inherit' | 'default' | 'large';
+
 type Props = {
   token: Token;
-  iconProps?: SvgIconProps;
+  iconSize?: IconSizes;
+  iconProps?: Omit<SvgIconProps, 'fontSize'>;
 };
 
-export function TokenName({ token, iconProps = {} }: Props) {
+export function TokenName({ token, iconSize = 'default', iconProps = {} }: Props) {
   const classes = useStyles();
   const { className, ...rest } = iconProps;
 
   return (
     <Grid container alignItems="center" wrap="nowrap">
-      <TokenIcon tokenAddress={token.address} className={cn(className, classes.icon)} {...rest} />
+      <TokenIcon
+        tokenAddress={token.address}
+        className={cn(className, classes.icon, getIconSizeClass(iconSize))}
+        {...rest}
+      />
       {token.symbol}
     </Grid>
   );
+
+  function getIconSizeClass(size?: IconSizes) {
+    return size && classes[size];
+  }
 }
 
 const useStyles = makeStyles(
   {
     icon: {
       marginRight: 8,
     },
+    default: {
+      fontSize: 24,
+    },
+    small: {
+      fontSize: 20,
+    },
+    large: {
+      fontSize: 30,
+    },
```

Comment:
SvgIcon theme overridden are: default = 20px, small = 16px, large = 24px. 
It is necessary to keep the default size same as in global styles to avoid adding iconSize="small" in most use cases in delphi.
Also, please check in delphi that fontSize="large" is replaced with iconSize="default".